rclcpp/include/rclcpp/client.hpp

```diff
@@ -93,11 +93,32 @@ class Client : public ClientBase
   RCLCPP_SMART_PTR_DEFINITIONS(Client);
 
   Client(
-    std::shared_ptr<rmw_node_t> node_handle,
-    rmw_client_t * client_handle,
-    const std::string & service_name)
-  : ClientBase(node_handle, client_handle, service_name)
-  {}
+    std::shared_ptr<rcl_node_t> node_handle,
+    const std::string & service_name,
+    rcl_client_options_t & client_options)
```

Comment:
You changed the order of the arguments here, does this not need changes in the examples and stuff like that?